modules/serverless-logic-upgrade-1-36-finalizing-job-service.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-upgrading-operator-from-1.35-to-1.36
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-logic-upgrade-1-36-finalizing-job-service_{context}"]
+= Finalizing the Job Service upgrade
+
+After the {ServerlessOperatorName} is upgraded to version 1.36.0 you can optionally delete the old `ReplicaSet` resource.
+
+.Procedure
+
+. (Optional) Identify the old `ReplicaSet` resource by running the following command:
++
+[source,terminal]
+----
+$ oc get replicasets -o custom-columns=Name:metadata.name,Image:spec.template.spec.containers[*].image -n <target_namespace>
+----
++
+.Example output
+[source,sql,subs="verbatim,quotes"]
+----
+Name                                                Image
+*sonataflow-platform-jobs-service-1111111111         registry.redhat.io/openshift-serverless-1/logic-jobs-service-postgresql-rhel8:1.35.0*
+sonataflow-platform-jobs-service-2222222222         registry.redhat.io/openshift-serverless-1/logic-jobs-service-postgresql-rhel8:1.36.0
+----
++
+. (Optional) Delete your old `ReplicaSet` resource by running the following command:
```

Comment:
The whole procedure is optional, but this step isn't optional if you want to do the procedure. I don't think that either of these steps should be marked optional

Reply:
This makes sense